app/scripts/views/complete_sign_up.js

```diff
@@ -94,9 +96,11 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
               return false;
             }
 
-            return self.getAccount().isSignedIn()
+            var account = self.getAccount();
+            return account.isSignedIn()
               .then(function (isSignedIn) {
                 if (isSignedIn) {
+                  self.interTabSend(EVENTS.SIGNIN_SUCCESS, account);
```

Comment:
FWIW, the logic of which screen to go to next is a mess. Now that we have the broker Behaviors, we can probably migrate some of this logic to the brokers and clean this up. Another PR.